decidim-meetings/app/packs/stylesheets/decidim/meetings/_item.scss

```diff
@@ -175,3 +175,9 @@
     }
   }
 }
+
+.iframe {
+  &_container_meetings {
+    @apply h-[60vh] w-full overflow-x-scroll;
```

Comment:
It seems that the problem is height not the width. 

![image](https://github.com/decidim/decidim/assets/105683/96a602de-f7cd-4be6-813f-4874e1b30f63)

We could merge it as it is, but we need to discuss with @decidim/product about how to handle it on tablet (various dimensions). We may need to use multiple breakpoints each one of it with own height.

Reply:
Hmmm that's strange. The ```overflow: scroll;``` seems to work pre-meeting, but not when you're about to enter. I'll revisit the issue once again, but yes lets discuss internally.

Reply:
@alecslupu What do you say about increasing the ```height``` a creating a default for the video conference container? As discussed privately, I have been successful in adding a scroll in the iframe of the meeting in DOM, but I can't seem to call the generated CSS class locally once a user enters a meeting via 8x8.

Reply:
@greenwoodt we could do some changes... As we already know the iframe domain name, we could add some additional css rules to the container: 

```
  <div class="iframe_container_meetings <%= helper_defined_by_us_to_return_the_class_name(meeting)%>">
    <%== embed_code(request.host) if show_embed? %>
  </div>
```

``` 
# helper ...
def helper_defined_by_us_to_return_the_class_name(meeting)
  case meeting.online_meeting_url 
    when /youtube/
       "youtube"
   when /twitch/ 
      "twitch"
    when /vimeo/
       "vimeo"
    default 
       ""
end
```

Reply:
@alecslupu it seems this method is still giving me a rails error no matter if I pass a different argument (meetings or method).

Reply:
@greenwoodt please add the following snippet to `decidim-meetings/app/cells/decidim/meetings/online_meeting_link_cell.rb` on line 18.

```
      def iframe_container_css_class
        @iframe_container_css_class ||= begin
          css_class = ["iframe_container_meetings"]

          case model.online_meeting_url
          when /youtube/
            css_class << "youtube"
          when /twitch/
            css_class << "twitch"
          when /meet\.jit\.si/
            css_class << "jitsi"
          when /vimeo/
            css_class << "vimeo"
            default
            ""
          end
          css_class.join(" ")
        end
      end
```

Adjust the template as follows: 

```
-  <div class="iframe_container_meetings">
-    <%== embed_code(request.host) if show_embed? %>
-  </div>
+    <div class="<%= iframe_container_css_class %>">
+      <%== embed_code(request.host) if show_embed? %>
+    </div>

```

This will output the following css classes to the container: `iframe_container_meetings jitsi` based on the jitsi link that i gave to you private.

Reply:
Understood. Will update this today!